http://forums.gentoo.org/viewtopic-p-6933728.html
http://lists.debian.org/debian-alpha/2011/11/msg00019.html
http://gcc.gnu.org/bugzilla/show_bug.cgi?id=47230 <- claims 2.18.50.0.9-8.fc10 20080822 works ok
http://lists.debian.org/debian-alpha/2011/11/msg00019.html
http://osdir.com/ml/linux.redhat.axp.general/2003-08/msg00039.html
http://sourceware.org/ml/binutils/2011-03/msg00525.html
http://sources.redhat.com/bugzilla/show_bug.cgi?id=230

init/built-in.o: In function `do_one_initcall':
linux-2.6.27.61/init/main.c:750: relocation truncated to fit: GPREL16 against `.sbss'
linux-2.6.27.61/init/main.c:757: relocation truncated to fit: GPREL16 against `.sbss'
linux-2.6.27.61/init/main.c:768: relocation truncated to fit: GPREL16 against `.sbss'
init/built-in.o: In function `init_post':
linux-2.6.27.61/init/main.c:840: relocation truncated to fit: GPREL16 against symbol `system_state' defined in .sbss section in init/built-in.o
linux-2.6.27.61/init/main.c:851: relocation truncated to fit: GPREL16 against `.sbss'
linux-2.6.27.61/init/main.c:863: relocation truncated to fit: GPREL16 against `.sbss'
linux-2.6.27.61/init/main.c:865: relocation truncated to fit: GPREL16 against `.sbss'
linux-2.6.27.61/init/main.c:853: relocation truncated to fit: GPREL16 against `.sbss'
init/built-in.o: In function `set_reset_devices':
linux-2.6.27.61/init/main.c:187: relocation truncated to fit: GPREL16 against symbol `reset_devices' defined in .sbss section in init/built-in.o
init/built-in.o: In function `unknown_bootoption':
linux-2.6.27.61/init/main.c:325: relocation truncated to fit: GPREL16 against `.sbss'
linux-2.6.27.61/init/main.c:333: additional relocation overflows omitted from the output
make: *** [.tmp_vmlinux1] Error 1

--- linux-2.6.27/arch/alpha/Makefile~	2008-10-09 22:13:53.000000000 +0000
+++ linux-2.6.27/arch/alpha/Makefile	2012-03-23 16:07:44.000000000 +0000
@@ -12,7 +12,7 @@
 
 LDFLAGS_vmlinux	:= -static -N #-relax
 CHECKFLAGS	+= -D__alpha__ -m64
-cflags-y	:= -pipe -mno-fp-regs -ffixed-8 -msmall-data
+cflags-y	:= -pipe -mno-fp-regs -ffixed-8 -mlarge-data
 cflags-y	+= $(call cc-option, -fno-jump-tables)
 
 cpuflags-$(CONFIG_ALPHA_EV4)		:= -mcpu=ev4

